app/templates/account/profile.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+	<div> 
+		{{ user.first_name }}
+	</div>
+	<div>
+		{{ user.last_name }}
+	</div>
+	<div>
+		{{ user.email }}
+	</div>
+	<div>
+		{{ user.role.name }}
+	</div>
+    <div>
+        {{ user.birthday }}
+    </div>
+	<div>
```

Comment:
this indenting is especially weird, make sure it's properly formatted. Check out this file for a good example: https://github.com/hack4impact/women-veterans-rock/blob/master/app/templates/admin/registered_users.html